src/index.js

```diff
@@ -41,41 +43,66 @@ const printList = () => {
   }
 
   element.innerHTML = tasks;
+
+  const confirmEditTask = (e) => {
+    const taskIndex = parseInt(e.target.parentNode.parentNode.id, 10);
+    const newDescription = document.getElementById(`edit${taskIndex}`).value;
+    const taskElement = document.getElementById(`${taskIndex}`);
+    const task = new Task(newDescription, taskIndex);
+    task.Edit();
+
+    document.querySelector(`#edit${taskIndex}`).classList.add('hidden');
+    document.querySelector(`#task${taskIndex}`).classList.remove('hidden');
+
+    taskElement.querySelector('.edit-manager').classList.add('hidden');
+    taskElement.querySelector('.list-editor').classList.remove('hidden');
+    printList();
+  };
+
+  // Start editting
   document.querySelectorAll('.edit-task').forEach((etb) => {
     etb.addEventListener('click', (e) => {
       const taskIndex = parseInt(e.target.parentNode.parentNode.id, 10);
+      document.getElementById(`edit${taskIndex}`).value = document.querySelector(`#task${taskIndex}`).innerHTML;
       const taskElement = document.getElementById(`${taskIndex}`);
-      // console.log(taskElement);
-
-      document.querySelector(`#edit${taskIndex}`).classList.remove('hidden');
+      const editTask = document.querySelector(`#edit${taskIndex}`);
+      editTask.classList.remove('hidden');
       document.querySelector(`#task${taskIndex}`).classList.add('hidden');
 
       taskElement.querySelector('.edit-manager').classList.remove('hidden');
       taskElement.querySelector('.list-editor').classList.add('hidden');
 
```

Comment:
[OPTIONAL]
- [ ] Editing also isn't working I am getting the same error as the delete error:
  ![image](https://user-images.githubusercontent.com/20567503/196301267-0eb703f8-82f4-41b4-82f2-9029cdcaf8ef.png)